.kitchen.yml

```diff
@@ -15,7 +15,7 @@ verifier:
   name: inspec
 
 platforms:
-  - name: a_centos_type
+  - name: centos/7
```

Comment:
Please revert this (but good catch on the quotes below)

Reply:
Good catch... darn it.